typescript/sdk/src/core/EvmCoreModule.ts

```diff
@@ -91,6 +96,11 @@ export class EvmCoreModule extends HyperlaneModule<
     transactions.push(
       ...(await this.createDefaultIsmUpdateTxs(actualConfig, expectedConfig)),
       ...this.createMailboxOwnerUpdateTxs(actualConfig, expectedConfig),
+      ...proxyAdminOwnershipUpdateTxs(
+        actualConfig,
+        expectedConfig,
+        this.domainId,
```

Comment:
another case of chainId/domainId conflation :D 

should use the chainId

Reply:
can you also have chainId be the first arg to be consistent with `transferOwnershipTransactions`